core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -244,4 +244,41 @@ private TableProperties() {
 
   public static final String UPSERT_MODE_ENABLE = "write.upsert.enable";
   public static final boolean UPSERT_MODE_ENABLE_DEFAULT = false;
+
+  //  compact small files
+  public static final String WRITE_AUTO_COMPACT_ENABLED = "write.auto-compact.enabled";
```

Comment:
Flink features like auto-compaction should include `flink` somewhere in the property name. Otherwise, people would think that this feature applies to other engines.